router/server.go

```diff
@@ -457,6 +432,34 @@ func (s *Server) forwardWebsocket(w http.ResponseWriter, req *http.Request, newu
 	wsServer.ServeHTTP(w, req)
 }
 
+/*
+forwardUDP does for UDP what forwardHTTP does for http requests. UDP is connectionless, so implementation is very simple
+*/
+func (s *Server) forwardUDP(w http.ResponseWriter, req *http.Request, newurl string) {
+
+	u, err := url.Parse(newurl)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	dstHost := u.Host // Destination address, e.g. 127.0.0.1:5984.
+	conn, err := net.Dial("udp", dstHost)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	defer conn.Close()
+
+	msg, err := ioutil.ReadAll(req.Body)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	conn.Write(msg)
+
+	fmt.Fprint(w, `OK`)
```

Comment:
Don't need this line, if you send anything back to the client it will send an 'OK' response by default.